### ISSUE TITLE
PDF: Changes in Table of Content

### ISSUE THREAD

**Issue by voisardf** [openoereb/pyramid_oereb#1213]
The new specifications introduces a third possible legal status (running modifications ?with anticipated effects?) on top of 'in force' and 'running modifications'. 
As soon as a real estate is concerned by any of these three cases, the topic has to be listed in the concerned topic list. If for a given topic only one or two of the law status concern the real estate, the status is omitted in the concerned topic list of the TOC but NOT displayed in the not concerned topic list (see also section 3.3.1 from the link below).

Details on pages 12 and 23 of the specification: https://github.com/openoereb/pyramid_oereb/files/6442522/Weisung-statischer-Auszug-de.pdf

### What needs to be done:
- [x] Change display logic of topics regarding their legal status
- [x] Move update date of the surveying data to the title page if not already done in #1212
- [x] Include information on restrictions potentially referenced in the land registry
- [x] Verify layout conformance (spacing, padding, dimensions, font sizes)
- [ ] Check for renamed attributes (TO DO: rename "exclusion of liability" to "disclaimer" when https://github.com/openoereb/pyramid_oereb/pull/1317/ is merged)
- [x] QR-Code -> see dedicated ticket #1270

**Comment by jwkaltz** [comment]
@voisardf regarding "Make sure QR-Code is backward compatible" - we don't have a QR-Code yet so we should reformulate "Add a QR-code", or create a dedicated issue for QR-Code? (I would prefer a dedicated issue, as it is a new functionality)

**Comment by voisardf** [comment]
Let's create a dedicate issue then. Thanks for the input!

**Comment by marionb** [comment]
@voisardf the link under the second point is broken. I found this one instead http://models.geo.admin.ch/V_D/OeREB/OeREBKRM_V2_0_Themen_20210714.xml is this correct?

Should we also create a point to remove the attachments in this issue? They were only used in the full extract but we should still remove them at one point as they are not used any more.

> Include information on restrictions potentially referenced in the land registry

This point is not clear to me. Do we already have this information somewhere in the templates? In what part of the TOC is it to be included?

**Comment by voisardf** [comment]
@marionb : yes that ist correct. Two of the mandatory files have been updated on July 14.

Not sure I understand you right. You mean get rid of the code allowing to create a full extract (with the attachment list on the TOC page)?

The point about the land registry is a disclaimer element coming from the http://models.geo.admin.ch/V_D/OeREB/ > OeREBKRM_V2_0_Texte_[update_date].xml:
`<OeREBKRMkvs_V2_0.Konfiguration.Haftungshinweis TID="43773.1">`
So it's a data issue, not a configuration issue and nothing needs to be done in the code as far as I can see.

**Comment by marionb** [comment]
@voisardf I guess I was not clear with my point:

> Not sure I understand you right. You mean get rid of the code allowing to create a full extract (with the attachment list on the TOC page)?

In the TOC we use two different templates depending on whether it is a full or reduced extract ([tocFull.jrxml](https://github.com/openoereb/pyramid_oereb_mfp/blob/master/print-apps/oereb/tocFull.jrxml) or [tocReduced.jrxml](https://github.com/openoereb/pyramid_oereb_mfp/blob/master/print-apps/oereb/tocReduced.jrxml)). This will no longer be needed. If we are touching the TOC templates here we should also remove the use of the full TOC template. Or is this done in another issue?

**Comment by voisardf** [comment]
@marionb Yes, we should clean up this part and no, it is not yet in an issue as far as I remember (@vvmruder: or am I wrong?).
So, yes please create an issue for that in the subproject related to the MFP adaption. 👍

**Comment by marionb** [comment]
@vvmruder, @voisardf and @jwkaltz For info here the new issue #1280 for cleaning up the full extract in MFP (should not be something too big).

**Comment by vuilleumierc** [comment]
@vvmruder do I understand correctly that the point
"Make sure the predefined titles OeREBKRM_V2_0_Themen_[update_date].xml are used for the mandatory topics"
is being taken care of in your PR https://github.com/openoereb/pyramid_oereb/pull/1317?

**Comment by voisardf** [comment]
@vuilleumierc : The correct titles for the topics must already be in the data extract. So that is not really an issue for MFP.

**Comment by vuilleumierc** [comment]
@voisardf OK, then I'll remove it from this issue

**Comment by vvmruder** [comment]
Yes, I try to solve it too in my PR. As François said, the titles need to be in the JSON/XML so MFP can expect to get correct values.

**Comment by marionb** [comment]
@voisardf @vvmruder and @vuilleumierc 
The sub-theme PR #1306 prepares the title that is sent to MFP to contain the sub-topic/sub-theme if there is one. This part of the title will be implemented once the PR is merged.
